lib/modules/commentQuickCollapse.js

```diff
@@ -35,3 +107,40 @@ function hideCommentsOnHeaderDoubleClick() {
 		}
 	});
 }
+
+function toggleCommentsOnClickLeftEdge() {
+	const leftEdgeWidth = _.clamp(parseInt(module.options.leftEdgeWidth.value, 10), 0, parseInt((module.options.leftEdgeWidth: any).default, 10)) / 100;
+	const stylesheet = toggleCommentsOnClickLeftEdgeStylesheetTemplate({
+		bColor: module.options.leftEdgeColor.value,
+		hColor: module.options.leftEdgeHoverColor.value,
+		cColor: module.options.leftEdgeCollapsedColor.value,
+		widthI: 2 * leftEdgeWidth,
+		widthO: 2.5 * leftEdgeWidth,
+	});
+
+	addCSS(stylesheet);
+}
+// scrolls on collapse
+function scrollOnCollapse() {
+	$(document.body).on('click', '.expand', (event: MouseEvent) => {
+		// Stops event if condition not met
+		if (click.isProgrammaticEvent(event) || Thing.checkedFrom(event.target).thing.classList.contains('noncollapsed')) {
+			return;
+		}
+		// Scrolls by a default value if top of comment is out of window
+		if ($(event.target).offset().top < $(window).scrollTop()) {
```

Comment:
We really ought to refactor all the KeyboardNav "move around in the comment tree" into its own library instead of calling an options callback like that.